### PR TITLE
Update manifest.yaml

### PR DESCRIPTION
**Pull request by dr-bonez**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -169,6 +165,14 @@ backup:
     io-format: yaml
 migrations:
   from:
+    "<0.14.2": # tor added in 0.14.2
```
Comment (chrisguida) [inline manifest.yaml:168]: Now there are multiple versions of LND that have multiple migrations defined.. what happens in that case? Does the first matching migration run? Or all matching migrations?

Comment (dr-bonez) [reply manifest.yaml:168]: first

**Review by chrisguida — APPROVED** [review]
Looks good 👍

**Comment by elvece** [comment]
**Upgrades**:
Uninstalled LND (to clear any remnants of new/old config keys) in between each test

- Installed 0.13.3.1 -> configured with bitcoin as internal ->  updated to 0.14.2.3 -> configured with new options (tor, advanced), bitcoin config remains core = success
- Installed 0.13.3.1 -> configured with bitcoin as external ->  updated to 0.14.2.3 -> configured with new options (tor, advanced), migrated bitcoin config from external to proxy (the default) = success

- Installed 0.13.3.2 -> configured with bitcoin as internal -> updated to 0.14.2.3 -> configured with new options (tor, advanced) = success
- Installed 0.13.3.2 -> configured with bitcoin as external -> updated to 0.14.2.3 -> configured with new options (tor, advanced), migrated bitcoin config from external to proxy (the default) = success

- Installed 0.14.2 -> configured with bitcoin as internal -> updated to 0.14.2.3 -> no configuration needed (bitcoin still internal core) = success
- Installed 0.14.2 -> configured with bitcoin as proxy-> updated to 0.14.2.3 -> no configuration needed (bitcoin still proxy) = success
- Installed 0.14.2 -> configured with bitcoin as external -> updated to 0.14.2.3 -> no configuration needed, bitcoin config migrated from external to proxy (default) = success

- Installed 0.14.2.1 -> configured with bitcoin as external -> updated to 0.14.2.3 -> no configuration needed, bitcoin config migrated from external to proxy (default) = success
- Installed 0.14.2.1 -> configured with bitcoin as internal -> updated to 0.14.2.3 -> no configuration needed, bitcoin config remained core  = success
- Installed 0.14.2.1 -> configured with bitcoin as proxy -> updated to 0.14.2.3 -> no configuration needed, bitcoin config remained proxy  = success

- Installed 0.14.2.2 -> configured with bitcoin as core -> updated to 0.14.2.3 -> no configuration needed, bitcoin config remained core = success
- Installed 0.14.2.2 -> configured with bitcoin as proxy -> updated to 0.14.2.3 -> no configuration needed, bitcoin config remained proxy = success
- Installed 0.14.2.2 -> configured with bitcoin as none/neutrino -> updated to 0.14.2.3 -> no configuration needed, bitcoin config remained none/neutrino = success

**Downgrades**:
Uninstalled LND (to clear any remnants of new/old config keys) in between each test

- Installed 0.14.2.3 -> configured with bitcoin as core-> downgraded to 0.13.3.1-> configured with new options (watchtower), bitcoin config remains internal = FE BUG*(Fixed link), MIGRATION BUG (fixed in this PR) = success
- Installed 0.14.2.3 -> configured with bitcoin as proxy -> downgraded to 0.13.3.1-> configured with new options (watchtower), migrated bitcoin config from proxy to internal (internal/external only options at this version) = FE BUG ([Fixed](https://github.com/Start9Labs/embassy-os/pull/1656)), MIGRATION BUG (Fixed in this PR) = success
- Installed 0.14.2.3 -> configured with bitcoin as none/neutrino -> downgraded to 0.13.3.1-> configured with new options (watchtower), migrated bitcoin config from none/neutrino to internal (internal/external only options at this version) = FE BUG*(Fixed link), MIGRATION BUG (Fixed link) = success


- Installed 0.14.2.3 -> configured with bitcoin as core ->  downgraded to 0.13.3.2 -> configured with new options (watchtower), bitcoin config remains internal = FE BUG*(Fixed link), MIGRATION BUG(fixed link) = success
- Installed 0.14.2.3 -> configured with bitcoin as proxy ->  downgraded to 0.13.3.2 -> configured with new options (watchtower), migrated bitcoin config from proxy to internal (internal/external only options at this version) = FE BUG ([Fixed](https://github.com/Start9Labs/embassy-os/pull/1656)) , MIGRATION BUG (fixed in this PR) = success
- Installed 0.14.2.3 -> configured with bitcoin as none/neutrino ->  downgraded to 0.13.3.2 -> configured with new options (watchtower), migrated bitcoin config from none/neutrino to internal (internal/external only options at this version) = FE BUG*([Fixed](https://github.com/Start9Labs/embassy-os/pull/1656)), MIGRATION BUG (fixed in this PR) = success


- Installed 0.14.2.3 -> configured with bitcoin as core -> downgraded to 0.14.2 -> no configuration needed (bitcoin remained internal) = success
- Installed 0.14.2.3 -> configured with bitcoin as proxy -> downgraded to 0.14.2 -> no configuration needed (bitcoin config remained proxy) = success
- Installed 0.14.2.3 -> configured with bitcoin as none/neutrino -> downgraded to 0.14.2 -> no configuration needed, migrated bitcoin config from none/neutrino to proxy (default) = success


- Installed 0.14.2.3 -> configured with bitcoin as proxy -> downgraded to 0.14.2.1 -> no configuration needed, bitcoin config remained proxy = success
- Installed 0.14.2.3 -> configured with bitcoin as core -> downgraded to 0.14.2.1 -> no configuration needed, bitcoin config remained none/neutrino) = success
- Installed 0.14.2.3 -> configured with bitcoin as none/neutrino -> downgraded to 0.14.2.1 -> no configuration needed, migrated bitcoin config from none/neutrino to proxy (default) = success


- Installed 0.14.2.3 -> configured with bitcoin as proxy-> downgraded to 0.14.2.2 -> no configuration needed, bitcoin config remained proxy = success
- Installed 0.14.2.3 -> configured with bitcoin as core -> downgraded to 0.14.2.2 -> no configuration needed, bitcoin config remained core = success
- Installed 0.14.2.3 -> configured with bitcoin as none/neutrino -> downgraded to 0.14.2.2 -> no configuration needed, bitcoin config remained none/neutrino = success


Other cases for record keeping:
Uninstalled LND
Installed 0.14.2 -> configured ->  downgraded to 0.13.3.2 -> configured with new options (watchtower) = success
Stayed on 0.13.3.2 -> downgraded to 0.13.2.1 -> drops into needs config, but no new options, this is expected as pre SLA = success
Uninstalled LND
Installed 0.14.2 -> configured with bitcoin as internal->  downgraded to 0.13.3.1 -> configured with new options (watchtower) = success
Stayed on 0.13.3.1 -> configured with bitcoin as external -> upgraded to 0.13.3.2 -> drops into needs config, but no new config options, expected as pre SLA, configured = success
Stayed on 0.13.3.2 (configured with bitcoin as external) -> upgraded to 0.14.2 -> configured with new options (tor, advanced) = success
Stayed on 0.14.2 -> upgraded to 0.14.2.1 -> drops into needs config, but no new options, expected as pre SLA , configured = success
Uninstalled LND
Installed 0.14.2.1 -> configured ->  downgraded to 0.13.3.2 -> configured with new options (watchtower) = success
Uninstalled LND
Installed 0.14.2.1 -> configured ->  downgraded to 0.13.3.1 -> configured with new options (watchtower) = success

**NOTE**: SLA determined in eOS 0.3.1 release that services should not drop into needs config unless there are new options

**NOTE**: It could happen that if you never configured a service in a prior version, then updated, it will not display that there are new config values available.

@chrisguida @dr-bonez @cryptodread @MattDHill

**Comment by dr-bonez** [comment]
This is very thorough, I'm impressed

**Comment by chrisguida** [comment]
Yeah wow

**Pull request merged**